### ISSUE TITLE
[Calamares] Module "partition@partition" exists but not loaded. 

### ISSUE THREAD

**Issue by oberon2007** [manjaro/packages-extra#83]
```
$ pkexec calamares
16:41:07 [1]: Calamares version: 2.2.2 
Translation: Calamares: Using system locale: "en_US"
Translation: Qt: Using default locale, system locale one not found: "en_US"
16:41:07 [1]: Default font ===== 
Pixel size:    -1 
Point size:    9 
Point sizeF:   9 
Font family:   "Sans Serif" 
Metric height: 14 
16:41:07 [1]: Font height: 14 
16:41:07 [1]: Available languages: ("ar", "ast", "bg", "ca", "cs_CZ", "da", "de", "el", "en", "en_GB", "es_MX", "es", "eu", "fr", "hu", "id", "is", "it_IT", "ja", "lt", "nl", "pl", "pt_BR", "pt_PT", "ro", "ru", "sv", "th", "tr_TR", "zh_CN", "zh_TW") 
16:41:07 [1]: CalamaresApplication thread: 0x128bab0 
16:41:07 [1]: Using log file: "/root/.cache/Calamares/Calamares/Calamares.log" 
16:41:07 [1]: Using Calamares settings file at "/usr/share/calamares/settings.conf" 
16:41:07 [1]: Using Calamares branding file at "/usr/share/calamares/branding/manjaro/branding.desc" 
16:41:07 [1]: WARNING: the selected branding component does not ship translations. 
16:41:07 [1]: Loaded branding component "manjaro" 
16:41:07 [1]: Proposed window size: 1010 520 
16:41:07 [1]: Initial locale  "en_US" 
16:41:07 [1]: ViewModule loading self for instance "welcome@welcome" 
ViewModule at address 0x13cbda0 
Calamares::PluginFactory at address WelcomeViewStepFactory(0x1350e10) 
ViewStep at address WelcomeViewStep(0x1338ae0) 
16:41:07 [1]: ViewModule loading self for instance "locale@locale" 
ViewModule at address 0x12d3fa0 
Calamares::PluginFactory at address LocaleViewStepFactory(0x148bac0) 
ViewStep at address LocaleViewStep(0x149f750) 
No layout variant specified on the command line
Trailing -variant option ignored
16:41:08 [1]: xkbmap selection changed to:  "us" - "" 
16:41:08 [1]: ViewModule loading self for instance "keyboard@keyboard" 
ViewModule at address 0x14a06c0 
Calamares::PluginFactory at address KeyboardViewStepFactory(0x148aa80) 
ViewStep at address KeyboardViewStep(0x15d08f0) 
16:41:08 [1]: virtual void Calamares::ViewModule::loadSelf() "Cannot load library /usr/lib/calamares/modules/partition/libcalamares_viewmodule_partition.so: (libkpmcore.so.2: cannot open shared object file: No such file or directory)" 
16:41:08 [1]: Module "partition" loading FAILED 
16:41:08 [1]: ViewModule loading self for instance "users@users" 
ViewModule at address 0x37eaa20 
Calamares::PluginFactory at address UsersViewStepFactory(0x37e88f0) 
ViewStep at address UsersViewStep(0x37c24c0) 
16:41:08 [1]: ViewModule loading self for instance "summary@summary" 
ViewModule at address 0x38148b0 
Calamares::PluginFactory at address SummaryViewStepFactory(0x3814ed0) 
ViewStep at address SummaryViewStep(0x3814dd0) 
16:41:08 [1]: Module "partition@partition" exists but not loaded. 
Calamares will now quit.
```
**calamares-dev** works fine.
Shall we switch packages in the iso-profiles, maybe? Or is the next **calamares** version coming already?

**Comment by philmmanjaro** [comment]
Well, I assume you run **unstable**. If you check a new **kpmcore** came out. **calamares-dev** is compiled against that lib. **calamares** not yet. I'm doing that right now. Would have been easier if you had spotted it yourself :smile:

**Comment by oberon2007** [comment]
> Would have been easier if you had spotted it yourself :smile:
hm ... next time for sure! :stuck_out_tongue_winking_eye:

**Comment by philmmanjaro** [comment]
Fixed with **v2.2.3-rc1**
